### DISCUSSION TITLE
Specify Organization Details in PDF?

### DISCUSSION THREAD

**Discussion by PaintedMithra** [PartialVolume/shredos.x86_64#307] [Ideas]
Absolutely loving ShredOS, but I was wondering if there was a way we could set the wiping organization details that would go into the PDF? Ideally we'd preconfig the wiping organization on the bootable media (ie, all wipes with that USB key would always populate the pdf), and it would query for the Customer Details. That would allow us to skip editing the PDF post-erase, which can be a real pain.

**Comment by PartialVolume** [comment]
At the drive selection screen, press c for config. You can edit organisation details and also add customer details to a simple database, both of which are saved to the USB stick when you exit nwipe (control c). Nwipe automatically reads host and customer details from the USB stick from /etc/nwipe/nwipe.conf and /etc/nwipe/customers.csv

If you are PXE booting you can also specify a FTP/tftp server where these files can be retrieved if you are not using a USB to boot ShredOS.

When you edit the organisation and/or customer details  after pressing the c key at the drive selection screen, the copies held on the virtual RAM disc are only saved to the USB stick or FTP/sftp server when exiting nwipe using control C (abort) or after a wipe has completed and you have pressed enter to generate the certificates and exit nwipe.

Hope that helps.

**Comment by PartialVolume** [comment]
>and it would query for the Customer Details.

When you go into c (config) there is an option to enable preview of organisation and customer details. If you enable this, then when nwipe starts, instead of showing the disc selection screen it will first show you the organisation and selected customer details. You can then edit organisation if necessary and add, delete and select from a list, the customer that appears on the PDF.

**Reply by PartialVolume** [reply]
On the organisation and customer preview screen, once you are happy with any changes you may make, press the A key to accept the organisation and customer details, the disc selection screen will then appear.

**Reply by PaintedMithra** [reply]
Well, ffs, I was sure I looked everywhere, but of course the answer is RIGHT IN THE FLIPPING program. /facepalm. Thanks so very much!
